src/client/wire_protocol/operations.rs

```diff
@@ -149,13 +192,13 @@ impl Message {
         let header = Header::with_query(total_length, request_id);
 
         Ok(Message::OpQuery { header: header, flags: flags,
-                              full_collection_name: full_collection_name,
+                              namespace: namespace,
```

Comment:
General comment: these changes are outside the scope of this commit. It's easy to get carried away at the beginning of the project, but let's work on making changes more focused going forward.